app/mailers/course_session_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class CourseSessionMailer < ApplicationMailer
+  def session_open_email
+    @course_session = params[:course_session]
+
+    mail(to: params[:user].email, subject: "The new session of Carrot U is open for applications")
```

Comment:
perhaps to avoid repetition with the body we say `Sign up for Carrot U` for the subject

Reply:
Good idea.